example/src/main/scala/example/HelloWorldAdvanced.scala

```diff
@@ -24,7 +24,8 @@ object HelloWorldAdvanced extends App {
   private val server =
     Server.port(PORT) ++              // Setup port
       Server.paranoidLeakDetection ++ // Paranoid leak detection (affects performance)
-      Server.app(fooBar ++ app)       // Setup the Http app
+      Server.app(fooBar ++ app) ++    // Setup the Http app
+      Server.lowLevelLogging          // Setup logging
```

Comment:
Maybe we can add performance warning comment here as well. Although not 100% sure, but sometimes the new users of a library refer to only examples rather than the documentation. 
```suggestion
      Server.lowLevelLogging          // Setup logging, degrades performance (only for low level debugging)
```